Week3/project3/project.js

```diff
@@ -0,0 +1,60 @@
+const url = "https://api.github.com/orgs/HackYourFuture/repos?per_page=100";
+  const HyfReposHttps = async ()=>{
+    const response = await axios(url);
+    return response;
```

Comment:
just wondering if it's possible to do `return await axios(url);` it's less code, it would be nice if you can try if it's going to work :D